charts/prometheus-smartctl-exporter/Chart.yaml

```
@@ -15,13 +15,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.3
+version: 0.4.4
```

```suggestion
version: 0.5.0
```


Thank you, @yellowhat. I'd suggest to raise the minor number instead.